src/client/utils/url-resolver.js

```diff
@@ -0,0 +1,59 @@
+import nativeMethods from '../sandbox/native-methods';
+import { ensureTrailingSlash } from '../../utils/url';
+import { get } from '../utils/destination-location';
+
+const DOCUMENT_URL_RESOLVER = 'hammerhead|document-url-resolver';
+
+function createResolver (doc) {
```

Comment:
This module should be an object and not the set of the utility methods. Since now it's single entity - `urlResolver`